### DISCUSSION TITLE
Invalid assert statements

### DISCUSSION THREAD

**Discussion by Lokejoke** [apache/airflow#44455] [General] [answered]
A new rule in Ruff exposed bugs in your code: literal is used as an assert message argument. Please refer to https://github.com/astral-sh/ruff/pull/14488#issuecomment-2488775594 if interested.

**Comment by potiuk** [comment]
NICE!

**Comment by potiuk** [comment, marked as answer]
Fixes in https://github.com/apache/airflow/pull/44460 -> thanks for letting us know @Lokejoke !
